topi/python/topi/x86/conv2d.py

```diff
@@ -651,9 +633,9 @@ def _declaration_conv_NCHWc_impl(cfg, data, kernel, kernel_size, strides, paddin
     kh = tvm.reduce_axis((0, kernel_size[0]), name='kh')
     kw = tvm.reduce_axis((0, kernel_size[1]), name='kw')
 
-    workload = conv_NCHWc_arg_to_workload(data, kernel, kernel_size,
-                                          strides, padding, layout,
-                                          out_layout, out_dtype),
+    workload = _conv_NCHWc_arg_to_workload(data, kernel,
```

Comment:
after using `autotvm.register_topi_compute`, we don't need to attach workload manully